www-misc/buku/buku-4.0.ebuild

```diff
@@ -9,26 +9,48 @@ inherit bash-completion-r1 distutils-r1
 
 DESCRIPTION="Powerful command-line bookmark manager"
 HOMEPAGE="https://github.com/jarun/Buku"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz -> ${P}.tar.gz"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/jarun/Buku"
+else
+	SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz -> ${P}.tar.gz"
+
+	KEYWORDS="~amd64 ~x86"
+fi
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
 IUSE="test"
 
 RDEPEND="
 	dev-python/beautifulsoup:4[${PYTHON_USEDEP}]
+	dev-python/certifi[${PYTHON_USEDEP}]
 	dev-python/cryptography[${PYTHON_USEDEP}]
+	dev-python/html5lib[${PYTHON_USEDEP}]
 	>=dev-python/urllib3-1.13.1[${PYTHON_USEDEP}]
+
```

Comment:
Unnecessary extra line, also please sort these deps in alphabetical order for better readability.

Reply:
buku have two programs buku and bukuserver. The first dependencies are of buku and the second set are of bukuserver, this way it's easier to keep track of bukuserver dependencies.

Reply:
Okay, I see. That's kind of smart, even though if both are shipped unconditionally, does it really matter in separating them? But wait for a dev to give their input. I've never seen RDEPEND list like this before, but it doesn't mean they don't exist! I'd personally probably try something like 
```
CLIENT_DEPEND="dev-python/beautifulsoup:4[${PYTHON_USEDEP}]
	dev-python/certifi[${PYTHON_USEDEP}]
	dev-python/cryptography[${PYTHON_USEDEP}]
	dev-python/html5lib[${PYTHON_USEDEP}]
	>=dev-python/urllib3-1.13.1[${PYTHON_USEDEP}]"
SERVER_DEPEND="dev-python/arrow[${PYTHON_USEDEP}]
	dev-python/click[${PYTHON_USEDEP}]
	dev-python/flask[${PYTHON_USEDEP}]
	dev-python/flask-admin[${PYTHON_USEDEP}]
	dev-python/flask-api[${PYTHON_USEDEP}]
	dev-python/flask-bootstrap[${PYTHON_USEDEP}]
	dev-python/flask-paginate[${PYTHON_USEDEP}]
	dev-python/flask-wtf[${PYTHON_USEDEP}]
	dev-python/requests[${PYTHON_USEDEP}]
	dev-python/werkzeug[${PYTHON_USEDEP}]"
RDEPEND="${CLIENT_DEPEND}
	${SERVER_DEPEND}"
DEPEND="${CLIENT_DEPEND}
	test? (
		dev-python/attrs[${PYTHON_USEDEP}]
		dev-python/flake8[${PYTHON_USEDEP}]
		dev-python/hypothesis[${PYTHON_USEDEP}]
		dev-python/py[${PYTHON_USEDEP}]
		dev-python/pytest[${PYTHON_USEDEP}]
		dev-python/pyyaml[${PYTHON_USEDEP}]
		dev-python/vcrpy[${PYTHON_USEDEP}]
	)"
```